store/src/java-test/com/zimbra/cs/account/MockProvisioning.java

```diff
@@ -56,6 +57,9 @@
  * @author ysasaki
  */
 public final class MockProvisioning extends Provisioning {
+    static {
+      LC.zimbra_attrs_directory.setDefault(System.getProperty("user.dir") + "/conf/attrs");
```

Comment:
Here, we are assuming that the user.dir will always be .../zm-mailbox/store
But that may not be the case for tests running from common or soap folder or from any zimbra extension like EWS
I think, System.getProperty("server.dir") should be used as we are doing in MailboxTestUtil.getZimbraServerDir(String zimbraServerDir)